index.js

```diff
@@ -74,9 +74,25 @@ function getData(swagger, path, operation, response, config, info) {
     queryApiKey: null,
     headerApiKey: null,
     headerSecurity: null,
-    path: ''
+    path: '',
+    isLoadTest: false,
+    requests: 0,
+    concurrent: 0
   };
 
+  // cope with loadTest info
+  if (info.loadTest != null) {
+    _.forEach(info.loadTest, function(loadTestParam) {
+      if (loadTestParam.pathName === path
+        && loadTestParam.operation === operation) {
+        info.importArete = true;
+        data.isLoadTest = true;
+        data.requests = loadTestParam.load.requests;
```

Comment:
Do we need to use a default value? I was thinking if the user does provide this, then we don't take care of it??